### PR TITLE
MLH-584 field age tests to check spinners (TCA-004, TCA-005)

### PR DESCRIPTION
**Pull request by LidiaLednev**

### CODE REVIEW

File: test/smoke/MLH-584_ageFieldSpinner_TCA.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+describe('MLH-584 field age tests to check spinners', function () {
+
+    it('TCA-004 Verify first click spinner Up.', function () {
+        browser.url('');
+        $$(sel.spinner)[0].click();
+        const Num1 = $(sel.age).getValue();
+        expect(Num1).toEqual("1");
+    });
+
+    it('TCA-005 Verify that spinner Up is present.', function () {
```
Comment (max9019) [inline test/smoke/MLH-584_ageFieldSpinner_TCA.js:12]: Test doesn't pass but spinner is present. Try adding moveTo() to point mouse over the age field first and then checking for spinner to present

File: test/smoke/MLH-584_ageFieldSpinner_TCA.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        browser.url('');
+        const spinnerUp = $$(sel.spinner)[0].isDisplayed();
+        expect(spinnerUp).toEqual(true);
+    });
+
+    it('TCA-005 Verify that spinner Down is present.', function () {
```
Comment (max9019) [inline test/smoke/MLH-584_ageFieldSpinner_TCA.js:20]: Test doesn't pass but spinner is present. Try adding moveTo() to point mouse over the age field first and then checking for spinner to present

**Review by max9019 — CHANGES_REQUESTED** [review]

File: test/smoke/MLH-584_ageFieldSpinner_TCA.js (hunk 2 above)
Comment (max9019) [reply test/smoke/MLH-584_ageFieldSpinner_TCA.js:20]: Still doesn't go through. Test shows there's no spinner displayed.

File: test/smoke/MLH-584_ageFieldSpinner_TCA.js (hunk 1 above)
Comment (max9019) [reply test/smoke/MLH-584_ageFieldSpinner_TCA.js:12]: Still doesn't go through. Test shows there's no spinner displayed.

**Review by max9019 — CHANGES_REQUESTED** [review]

File: test/smoke/MLH-584_ageFieldSpinner_TCA.js (hunk 2 above)
Comment (LidiaLednev) [reply test/smoke/MLH-584_ageFieldSpinner_TCA.js:20]: I  used moveTo(), and it didn't work. I don't understand how to  point mouse over the age field

**Review by max9019 — APPROVED** [review]

**Pull request merged**